adapter.go

```diff
@@ -65,6 +67,14 @@ func OpDatabaseName(dbName string) func(*adapter) {
 	}
 }
 
+// OpDatabaseCreds configures username and password of database used; default is ""
+func OpDatabaseCreds(user, passwd string) func(*adapter) {
```

Comment:
As different types of authentication are possible it would be nice to explicitly state type in name, eg. `OpBasicAuthCredentials()`.

Reply:
fixed